src/nuklear_button.c

```diff
@@ -78,6 +78,8 @@ nk_button_behavior(nk_flags *state, struct nk_rect r,
 #else
                 nk_input_is_mouse_pressed(i, NK_BUTTON_LEFT);
 #endif
+			if (ret)
```

Comment:
What about it?

Reply:
There are 3 tabs on that line, but `nuklear.h` uses spaces for indentation. At best, we should stick with that instead of introducing inconsistencies.

Reply:
Fixed, I think.